declarations/Bardeen.history.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Privacy Policy": [
+    {
+      "fetch": "https://www.bardeen.ai/privacy-policy",
+      "select": [
+        ".content-text"
+      ],
+      "validUntil": "2024-07-03T06:30:00Z"
```

Comment:
How do you get this `validUntil`date?
Following our [documentation](https://docs.opentermsarchive.org/terms/declarations-maintenance/#how-to-get-the-validuntil-date-from-an-issue) i find `2024-07-03T06:30:07Z`

Reply:
My copy-pasting mangled the last digits for some reason and I just thought the seconds don't really matter… If you think it's important that it matches, okay to make it match!

Reply:
The only thing that worries me about this value being exact is the fact that it could have an impact in the case of a history rewrite, but I'm not clear enough on how this could or could not be important.